infrastructure/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google"
+      version = "3.86.0"
+    }
+  }
+  # Run locally
+  # backend "gcs" {
+  #   credentials = "key.json"
+  #   bucket      = "titus-terraform-state"
+  #   prefix      = "core"
+  # }
+}
+
+# The service account used to run Terraform requires the:
+#  - Project Owner role
+provider "google" {
+  credentials = file("service-account-credentials.json")
+  project     = var.gcp_project_id
+  region      = var.region
+  zone        = var.zone
+}
+
+provider "google-beta" {
```

Comment:
Just wondering if this second provider block is actually needed as it is a duplicate of the one above?

Reply:
👍